packaging/Makefile

```diff
@@ -15,7 +15,7 @@ debian-package-code:
 	cp ../examples/instance_message_from_aleph.json ./aleph-vm/opt/aleph-vm/examples/instance_message_from_aleph.json
 	cp -r ../examples/data ./aleph-vm/opt/aleph-vm/examples/data
 	mkdir -p ./aleph-vm/opt/aleph-vm/examples/volumes
-	pip3 install --target ./aleph-vm/opt/aleph-vm/ 'aleph-message==0.4.1' 'jwskate==0.8.0' 'eth-account==0.9.0' 'sentry-sdk==1.31.0' 'qmp==1.1.0'
+	pip3 install --target ./aleph-vm/opt/aleph-vm/ 'aleph-message@git+https://github.com/aleph-im/aleph-message.git@mhh-pay-as-you-go-v1#egg=aleph-message' 'jwskate==0.8.0' 'eth-account==0.9.0' 'sentry-sdk==1.31.0' 'qmp==1.1.0' 'superfluid@git+https://github.com/Godspower-Eze/superfluid.py#egg=superfluid' 'sqlalchemy[asyncio]' 'aiosqlite==0.19.0' 'alembic==1.13.1'
```

Comment:
We already install `alembic` using the Debian packages, see `packaging/aleph-vm/DEBIAN/control`. This would create a conflict of versions. [Debian 10 alembic package](https://packages.debian.org/bullseye/alembic). I am open to only use the vendored version if it brings enough value compared to the one from the distribution.

Reply:
Moved to a separate issue.